components/basecomponents/StoryText.tsx

```diff
@@ -18,57 +19,71 @@ const scrollToRef = (ref: MutableRefObject<HTMLParagraphElement | null>, div: Mu
   }
 };
 
-const StoryText = ({ textLanguage, audioLanguage, id, audio, onClick }: StoryTextProps): JSX.Element => {
+const getPosition = (phrase: StoryPhrase, audioLanguage: Language): { end: number; start: number } => {
+  type ObjectKey = keyof typeof phrase;
+  const audioLanguageKey = audioLanguage === 'uk' ? 'uk' : 'cs';
+  const start = `start_${audioLanguageKey}` as ObjectKey;
```

Comment:
Try to avoid typecasting with `as` as much as possible. In this case you don't need, you only need to declare the strings `as const`. 

```ts
  const audioLanguageKey = audioLanguage === 'uk' ? 'uk' : 'cs';
  const start = `start_${audioLanguageKey}` as const;
  const end = `end_${audioLanguageKey}` as const;

  return { start: Number(phrase[start]), end: Number(phrase[end]) };
 ```
 
 In your version  you could pass any string to `start`. With `as const` you can only pass `start_cs` or `start_uk`.